### PR TITLE
Can update profile and password now

### PR DESCRIPTION
**Pull request by kfr7**

### CODE REVIEW

File: motivateme-ui/src/components/SecurityChange/SecurityChange.jsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+  const handleOnPasswordObjectChange = (event) => {
+    setSuccessMessage(null);
+    if (passwordError) {
+      setPasswordError(null);
+    }
+
+    if (event.target.name === "newPassword") {
+      if (passwordObject.confirmNewPassword !== event.target.value) {
+        console.log("entered don't match");
```
Comment (LifeBringer) [inline motivateme-ui/src/components/SecurityChange/SecurityChange.jsx:35]: Comment out/delete

File: motivateme-ui/src/components/SecurityChange/SecurityChange.jsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        setPasswordError("The new passwords do not match");
+      } else {
+        console.log("entered OKAY");
```
Comment (LifeBringer) [inline motivateme-ui/src/components/SecurityChange/SecurityChange.jsx:38]: Comment out/delete

File: motivateme-ui/src/components/SecurityChange/SecurityChange.jsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+        setPasswordError(null);
+      }
+    }
+    if (event.target.name === "confirmNewPassword") {
+      if (passwordObject.newPassword !== event.target.value) {
+        console.log("entered don't match");
```
Comment (LifeBringer) [inline motivateme-ui/src/components/SecurityChange/SecurityChange.jsx:45]: Comment out/delete

File: motivateme-ui/src/components/SecurityChange/SecurityChange.jsx [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+        setPasswordError("The new passwords do not match");
+      } else {
+        console.log("entered OKAY");
```
Comment (LifeBringer) [inline motivateme-ui/src/components/SecurityChange/SecurityChange.jsx:49]: Comment out/delete

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See comments

File: motivateme-ui/src/components/SecurityChange/SecurityChange.jsx (hunk 4 above)
Comment (kfr7) [reply motivateme-ui/src/components/SecurityChange/SecurityChange.jsx:49]: Will do.

**Review by StaticMaster — APPROVED** [review]
LGTM

**Pull request merged**